docs/modules/integrations/pages/github-action/upload-an-app.adoc

```diff
@@ -0,0 +1,18 @@
+= Upload an app to Kobiton
+:navtitle: Upload an app
+
+Learn how to upload an app to your app repository using GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#setup-workflow-and-action[Official GitHub Marketplace Kobiton Action].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].

```

Comment:
Remove this line as it is kind of the same with the second bullet point.

Reply:
1st bullet point " A Kobiton account. To create a Kobiton account visit link:[https://portal.kobiton.com/register[Kobiton]](https://portal.kobiton.com/register%5BKobiton%5D)." removed.